app/src/main/java/com/henagle/www/kanban/DeckAdapter.java

```diff
@@ -0,0 +1,82 @@
+package com.henagle.www.kanban;
```

Comment:
Moved this class out from the `DeckFragment.java` file because it was getting large